code/__DEFINES/economy.dm

```diff
@@ -1,4 +1,5 @@
 #define STARTING_PAYCHECKS 7
+#define MAX_MAIL_PER_MIN 3
```

Comment:
1. MINUTE (especially since `min` has dual meaning)
2. Doc comment it (since this is a global)